src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -156,6 +158,26 @@ internal void EnsureInitialized(WebHostSettings settings)
             }
         }
 
+        internal static WebHostSettings CreateStandbySettings(WebHostSettings settings)
+        {
+            // we need to create a new copy of the settings to avoid modifying
+            // the global settings
+            // important that we use paths that are different than the configured paths
+            // to ensure that placeholder files are isolated
+            string tempRoot = Path.GetTempPath();
+            var standbySettings = new WebHostSettings
+            {
+                LogPath = Path.Combine(settings.LogPath, "Standby"),
```

Comment:
why not temp folder? we cannot read it anyway if its put it in `D:\home`
temp folder is also faster

Reply:
Good question. Initially I was trying to keep things under the standard Log directory to preserve Kudu log streaming. However, that isn't needed for placeholder sites - we're fine with system logs which we'll still get.